Test/TestFramework/Shared/StubTelemetryInitializer.cs

```diff
@@ -19,7 +20,7 @@ public StubTelemetryInitializer()
         /// <summary>
         /// Gets or sets the callback invoked by the <see cref="Initialize"/> method.
         /// </summary>
-        public TelemetryAction OnInitialize { get; set; }
+        public Action<ITelemetry> OnInitialize = item => { };
```

Comment:
I'm not familiar with this syntax, is this saying the default getter returns item but there is an empty method for the setter?
Why not OnInitialize { get; } = item;?